src/emulator/functionsEmulator.ts

```diff
@@ -141,14 +140,14 @@ export class FunctionsEmulator implements EmulatorInstance {
       req: express.Request,
       res: express.Response
     ) => {
-      this.handleBackgroundTrigger(req, res);
+      await this.handleBackgroundTrigger(req, res);
```

Comment:
Why this change? Also, I'll save the bikeshedding about why async/await stuff here is misleading, but you get the idea.

Reply:
Oh yeah I need to revert this.  This was me messing around while investigating.